crates/mun_hir/src/ty.rs

```diff
@@ -136,6 +136,18 @@ impl Ty {
             _ => None,
         }
     }
+
+    pub fn ty_to_string(&self, db: &impl HirDatabase) -> Option<String> {
```

Comment:
there already is something to do this `ty.display(db)`  will create a formatter.